src/Components/Facility/AddInventoryForm.tsx

```diff
@@ -10,6 +12,7 @@ import {
   getInventorySummary,
 } from "../../Redux/actions";
 import * as Notification from "../../Utils/Notifications.js";
+import Page from "../Common/components/Page";
 import {
   LegacySelectField,
   LegacyTextInputField,
```

Comment:
You've not replaced any of the legacy components. Replace them with equivalent FormField components. Have a look at other similar PRs.

Reply:
now i replace legacy component